bundles/AdminBundle/Controller/Searchadmin/SearchController.php

```diff
@@ -77,9 +77,9 @@ public function findAction(Request $request, EventDispatcherInterface $eventDisp
 
         $query = $this->filterQueryParam($allParams['query'] ?? '');
 
-        $types = explode(',', $allParams['type'] ?? '');
-        $subtypes = explode(',', $allParams['subtype'] ?? '');
-        $classnames = explode(',', $allParams['class'] ?? '');
+        $types = explode(',', preg_replace("/[^a-zA-z,]/", '', $allParams['type'] ?? ''));
+        $subtypes = explode(',', preg_replace("/[^a-zA-z,]/", '', $allParams['subtype'] ?? ''));
+        $classnames = explode(',', preg_replace("/[^a-zA-z,]/", '', $allParams['class'] ?? ''));
```

Comment:
```suggestion
        $types = explode(',', preg_replace("/[^a-zA-Z,]/", '', $allParams['type'] ?? ''));
        $subtypes = explode(',', preg_replace("/[^a-zA-Z,]/", '', $allParams['subtype'] ?? ''));
        $classnames = explode(',', preg_replace("/[^a-zA-Z,]/", '', $allParams['class'] ?? ''));
```

Reply:
thanks :)

Reply:
Or maybe:

```suggestion
        $types = explode(',', preg_replace("/[^a-z,]/i", '', $allParams['type'] ?? ''));
        $subtypes = explode(',', preg_replace("/[^a-z,]/i", '', $allParams['subtype'] ?? ''));
        $classnames = explode(',', preg_replace("/[^a-z,]/i", '', $allParams['class'] ?? ''));
```

Reply:
`/i` simplifies the regex - I think that would be the best solution - thanks (again) :)